k8s/aat/common/ccpay/fees-register-frontend.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fees-register-frontend
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: fees-register-frontend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: fees-register-frontend
+    version: 0.1.1
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/fees-register/frontend:latest
+      ingressHost: adoption.aat.platform.hmcts.net
```

Comment:
may be we need to change ingress host here,  it should be fees-register.{{ .Values.global.environment }}.platform.hmcts.net , or fees-register.aat.platform.hmcts.net

Reply:
lol, thanks